premium/usage.go

```diff
@@ -226,7 +230,34 @@ func (u *BatchUpdater) Increase(rows uint32) error {
 		return fmt.Errorf("usage updater is closed")
 	}
 
-	u.rowsToUpdate.Add(rows)
+	u.mutex.Lock()
+	defer u.mutex.Unlock()
+	u.rows += rows
+
+	// Trigger an update unless an update is already in process
+	select {
+	case u.triggerUpdate <- struct{}{}:
+	default:
+		return nil
+	}
+
+	return nil
+}
+
+func (u *BatchUpdater) IncreaseWithTableBreakdown(table string, rows uint32) error {
```

Comment:
```suggestion
func (u *BatchUpdater) IncreaseForTable(table string, rows uint32) error {
```

I would suggest naming it something like this, since a breakdown (at least to me) would imply multiple tables being passed in to this function

Reply:
That's what I initially started with, but then I went with this approach as there are a lot of functions exposed by the SDK that go for the `With` prefix, so I though I should not stray from that convention.
Wdyt @hermanschaaf, should I go ahead and rename to `IncreaseForTable` or leave it as it is?

Reply:
`With` is often used for function Options, I don't think there's a hard rule it has to be used for function names other than in that case. I would say let's use a name that fits, and to me "breakdown" doesn't seem to fit here

Reply:
Done